src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -0,0 +1,970 @@
+# coding=utf-8
+# Copyright 2022 HuggingFace Inc. team and BigScience workshop.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""PyTorch BLOOM model."""
+
+import math
+from typing import Tuple
+
+import torch
+import torch.nn.functional as F
```

Comment:
we don't use `F`, please remove this import and switch to `nn.functional` in the caller space. `nn` is already imported.

@sgugger, would you consider adding to your magical validation function this case?

And also `torch.nn` uses, which shouldn't be there either?

Otherwise all the efforts that were done to normalize the consistent use of torch APIs across models is getting spoiled. 

Thank you!

Reply:
Fixed in a3741d421b3c804089be5696eb30dd4c338a7505 !